GitHubSearch/Services/Realm/RealmService.swift

```diff
@@ -17,7 +17,7 @@ struct RealmService {
         Realm.Configuration.defaultConfiguration = realmConfig
     }
 
-    private var realm: Realm {
+    var realm: Realm {
```

Comment:
We should keep it `private` as well as the `write()` method, because we use it to work with Realm inside of the `RealmSevice`